### ISSUE TITLE
[FEATURE]: Adding A workflow insuring including JSDOCS on every PR

### ISSUE THREAD

**Issue by mohmmadAyesh** [TheAlgorithms/JavaScript#1653]
### Motivation

Hi I thought it would be a nice idea to add a workflow that check if there is JSDOCS for every function when PR is opened do you want this feature in repo I will try to added if you accept

**Comment by appgurueu** [comment]
Sounds useful. But there need not be a doc comment for every function, just for *exported* functions. It might very well be reasonable not to have doc comments for small internal helpers.

**Comment by mohmmadAyesh** [comment]
well workflow it can be made only for exported functions but for small internal helpers dont know what criteria about how you can specify its a small helper file that doesnt need any JSDOCS

**Comment by appgurueu** [comment]
> well workflow it can be made only for exported functions

Yes. That's what I'm asking for. There should be no checks for internal functions.
